### PR TITLE
instructions for installing from .deb packages

### PR DESCRIPTION
**Pull request by eastlondoner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,28 @@ The library supports multiple versions of the Bolt protocol through the new _Con
...
+```
+sudo apt-get install -y libssl1.0.0
+```
+
+2. Fetch the latest package (check [here](https://github.com/neo4j-drivers/seabolt/releases) for latest releases)
+```
+wget https://github.com/neo4j-drivers/seabolt/releases/download/v1.7.0/seabolt-1.7.0-Linux-Ubuntu-Xenial.deb
+```
+
+3. (optional) Check the signature
```
Comment (ali-ince) [inline README.md]: We currently don't sign anything at all - would it make more sense to remove 3 all? wdyt?

**Review by ali-ince — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (eastlondoner) [reply README.md]: hmm 
A) we ought to sign stuff
B) do we at least publish a SHA hash that people can check?

Comment (ali-ince) [reply README.md]: I believe we'll only start signing stuff when we remove the `experimental` status on binary packages and maybe when we start publishing them to a repository.

There are SHA256 checksums available at https://github.com/neo4j-drivers/seabolt/releases along with the assets themselves.

Comment (ali-ince) [reply README.md]: and also could you please rebase the PR?

**Pull request merged**